src/pages/DetailsPage.jsx

```diff
@@ -1,15 +1,28 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Heading from "../components/Heading";
 import Breadcrumbs from "../components/Breadcrumb";
 import Input from "../components/Input";
 import Select from "../components/Select";
 import CurrentDate from "../components/Date";
 
 function DetailsPage() {
-  const options = [
-    { currency: "elo", label: "elo" },
-    { currency: "siema", label: "siema" },
-  ];
+  const [list, setList] = useState("");
+
+  useEffect(() => {
+    fetch(
+      ` https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest/currencies/eur.json`,
```

Comment:
Small remark, create a constants directory and store the base URL there. 

in `constants/baseUrl.js`:

```js
const BASE_URL = "https://cdn.jsdelivr.net/gh/fawazahmed0/currency-api@1/latest"
```

then here: 

```suggestion
      `${baseUrl}/currencies/eur.json`,
```

Reply:
done :)

Reply:
done